File: Maximum Ice cream bars.cpp
class Solution {
public:
    int maxIceCream(vector<int>& costs, int coins) {
        sort(costs.begin(),costs.end());
        int count=0;
        for(int i=0;i<costs.size();i++)
        {
            if(costs.at(i)<=coins)
            {
             coins=coins-costs.at(i);
             count++; 
            }
            else 
            break;

        }
     return count;
    }
};

File: progress sheet/maximum-ice-cream-bars.py
class Solution:
    def maxIceCream(self, costs: List[int], coins: int) -> int:
        costs.sort()
        count=0
        i=0
        while i < len(costs):
            if costs[i]>coins:
                 break
            coins-=costs[i]
            count=count+1
               
            i=i+1
        return count

